src/irr/asset/CGeometryCreator.cpp

```diff
@@ -8,13 +8,16 @@
 #include "irr/asset/normal_quantization.h"
 #include "irr/asset/CCPUMesh.h"
 
+#define NEW_SHADERS
```

Comment:
@AnastaZIuk GeometryCreator will need a large refactor

return type should be `ICPUMeshBuffer` with an `ICPURenderpassIndependentPipeline` member that is slightly invalid (no shaders, arbitrary `SBlendParams` and `SRasterizationParams`)

Reply:
@AnastaZIuk have you seen this comment?

Reply:
No I haven't, but I'm gonna see it ASAP

Reply:
@devshgraphicsprogramming okay, I have already read whole PR
GeometryCreator needs refactoring and meshbuffers in certain loaders need non-null pipeline
I will deal with PLY and X loader and then gonna fill them up

Reply:
first fix master on the old pipeline.

Reply:
Sure